lib/exposes.js

```diff
@@ -325,6 +327,7 @@ module.exports = {
         illuminance: () => new Numeric('illuminance', access.STATE).withDescription('Raw measured illuminance'),
         illuminance_lux: () => new Numeric('illuminance_lux', access.STATE).withUnit('lx').withDescription('Measured illuminance in lux'),
         keypad_lockout: () => new Lock().withState('keypad_lockout', '1', '0', 'Enables/disables physical input on the device'),
+        led_disabled: () => new Binary('led_disabled', access.STATE, true, false).withDescription('Indicates whether the device has LED disable at night state or not'),
```

Comment:
I think we should rename this to `led_disabled_night` because `led_disabled` makes me think the led is always disabled.

Reply:
Accepted